packages/itwin/grouping-mapping-widget/src/widget/components/context/GroupHilitedElementsContext.ts

```diff
@@ -13,7 +13,7 @@ export interface GroupHilitedElements {
   groups: Group[];
   setGroups: (groups: Group[]) => void;
   setHiddenGroupsIds: (hiddenGroupIds: string[]) => void;
-  setShowGroupColor: (showGroupColor: boolean) => void;
+  setShowGroupColor: (showGroupColor: boolean | ((showGroupColor: boolean) => boolean)) => void;
```

Comment:
Setter should only accept a boolean as an argument, not a function, which accepts a boolean argument.